### PR TITLE
Add serverless function

### PR DESCRIPTION
**Pull request by githinho**

### CODE REVIEW

**Comment by mightymatth** [comment]
@githinho this link should be working:
https://amptzr-git-sd-338-create-frontend-api-ampnetx.vercel.app/api/ipfs/issuer?hash=QmYa7ysVq5594LiwxVmadoDDSumrf15sBpSnSLrML9j2Hy

**Comment by githinho** [comment]
@mightymatth looks good, ready to merge

File: api/ipfs/issuer.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const httpClient = new HttpClient(new HttpXhrBackend({build: () => new XMLHttpRequest()}))
+const ipfsService = new IpfsService(httpClient)
+
+export default async (request: VercelRequest, response: VercelResponse) => {
+  const {hash} = request.query
+
+  return ipfsService.get<IPFSIssuer>(hash as string).pipe(
+    tap(issuer => response.status(200).send(issuer)),
+    catchError(_err => {
+      response.status(200).send({})
```
Comment (mightymatth) [inline api/ipfs/issuer.ts:19]: @githinho, in case of error, is this good behavior, or do you want 400? how do you like it?

Comment (githinho) [reply api/ipfs/issuer.ts:19]: I would rather return 502 - https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/502
Do you know which errors could accrue from ipfs?

Comment (githinho) [reply api/ipfs/issuer.ts:19]: Leave the empty response.

**Pull request merged**